src/languages/sqf.js

```diff
@@ -1884,7 +1999,7 @@ export default function(hljs) {
     'setCurrentTask',
     'setCurrentWaypoint',
     'setCustomAimCoef',
-    'setCustomMissionData',
+    'SetCustomMissionData',
```

Comment:
```suggestion
    'setCustomMissionData',
```
https://community.bistudio.com/wiki/setCustomMissionData